app/javascript/src/components/common-prop-types/article-prop-types.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
```

Comment:
Create the common article [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html).